blueocean-dashboard/src/main/js/components/PipelineRunGraph.jsx

```diff
@@ -173,7 +182,7 @@ export default class PipelineRunGraph extends Component {
 PipelineRunGraph.propTypes = {
     pipelineName: string,
     branchName: string,
-    runId: string,
```

Comment:
:ant: Then you should remove the attribute in RunDetailsPipeline.jsx as well in the Extensions.Renderer